rpa-bot/1.1/modules/ROOT/pages/configuration.adoc

```diff
@@ -0,0 +1,137 @@
+= Configuring RPA Bot
+
+Use the RPA Bot Configurator application to change the default settings of a bot. You can find the application in the start menu under *MuleSoft* > *RPA Bot Configurator*.
+
+The RPA Bot Configuration tool has five different configuration panels:
+
+* <<common-settings, Common>>
+* <<proxy-settings, Proxy>>
+* <<secure-session, Secure Session>>
+* <<connection-settings, Connection Settings>>
+* <<session-monitoring, Session Monitoring>>
+
+[TIP]
+After saving your changes, restart the RPA Bot Service for the changes to take effect.
+
+[[common-settings]]
+== Common Settings
+
+image:bot-configurator-common.png[The RPA Bot configurator application showing the Common settings panel, 50%, 50%]
+
+* *RPA Bot ID*
++
+ID generated during the installation process of RPA Bot, which identifies the bot in RPA Manager (even if the IP or MAC-Address changes). If you need a new ID, uninstall and then reinstall RPA Bot.
+* *RPA Manager to connect*
++
+The URL address of the RPA Manager instance the bot connects to. There is no default setting.
+* *Use HTTPS as protocol layer*
++
+Specifies whether the bot connects with RPA Manager via HTTP or HTTPS.
+* *API Key*
++
+The API Key entered during the installation of the robot. You can change this setting.
+* *Alive interval*
++
+Time interval in milliseconds after which the bot connects to the RPA Manager to show that it is still alive.
+* *Check interval*
++
+Time interval in milliseconds after which the bot connects to RPA Manager and checks for provided packages to collect (such as Scheduling Tasks or Activities).
+* *Cleanup workflows on start*
++
+Select this option to delete the workflows that are no longer assigned to a configuration but are still stored locally in the bot.
+
+[[proxy-settings]]
+== Proxy Settings
+
+image:bot-configurator-proxy.png[The RPA Bot configurator application showing the Proxy settings panel, 50%, 50%]
+
+* *Proxy*
++
+The bot and RPA Manager communicate through the HTTP protocol at Port 80. If you
+connect to the internet through a proxy server, enter the proxy address in this field
++
+Format: `+http://<Serveraddress>:<Port>+`, for example: `+http://myproxy.de:881+`.
+
+[[secure-session]]
+== Secure Session
+
+Enable the bot to use Secure Sessions by entering credentials the bot uses to login to the computer and to open a hidden session. If you don't add credentials here, the bot can only use a running desktop session.
+
+image:bot-configurator-secure-session.png[The RPA Bot configurator application showing the Secure Session settings panel, 50%, 50%]
+
+* *Default Secure Session User*
++
+If you want the bot to use Secure Sessions, enter the name of the user the bot uses to login to the computer and to run a hidden session. If the environment of the computer the bot is installed on is embedded in a domain and you want to use a local user, write the user name as follows: `.\username`.
```

Comment:
Please add an example for Domain users. They are often defined in the form of username@mydomain.com
This is not usable here. So the user must enter such a domain user in this form:  mydomain\username
The TLD (.com) must be omitted.

BTW: In one of the next versions we will show an hint in the dialog if the user tries to use the @ mail format.